src/main.ts

```diff
@@ -99,6 +99,25 @@ async function upload_to_release(
   return uploaded_asset.data.browser_download_url
 }
 
+function repo(): {owner: string; repo: string} {
+  const name = core.getInput('repo_name')
+  if (!name) {
+    return github.context.repo
+  }
+  const owner = name.substr(0, name.indexOf('/'))
```

Comment:
Wouldn't it perhaps be advisable to use octokit here (which we have in the context anyway) to do this operation a bit more safely?

Reply:
@svenstaro, can you suggest a concrete `Octokit` API I should use for converting the `the-owner/the-repo-name` string into `{ owner: 'the-owner', repo: 'the-repo-name' }` object?

If I'm not mistaken, the [GH Actions code does nothing more than splitting](https://github.com/actions/toolkit/blob/4f7fb6513a355689f69f0849edeb369a4dc81729/packages/github/src/context.ts#L59) the `owner` and `repo_name` from the string. Please advise how you want to ensure more safety. Thanks!

Reply:
Eh, I suppose if it's good enough for GitHub, it's good enough for us.